scripts/beck/st/error_injection.py

```diff
@@ -523,7 +523,6 @@ def ListAllEmapPerDevice():
         _, _, device_id = ConvertAdstob2Cob(stob_f_container_hex, stob_f_key_hex)
         # 16 bytes of BE_EMAP_KEY (footer) + 16 bytes of BE_EMAP_REC(header)
         # gives offset of Corresponding BE_EMAP_REC
-        emap_rec_offset = offset + 32
         # emap_rec_data, _ = ReadCompleteRecord(emap_rec_offset)
```

Comment:
Please remove this commented code as well.

Reply:
done